sql-plugin/src/main/scala/com/nvidia/spark/rapids/TypeChecks.scala

```diff
@@ -776,6 +776,23 @@ abstract class TypeChecks[RET] {
     }.mkString(", ")
   }
 
+  private def tagCustomMessage(
+    groupedByType: Map[DataType, Set[String]],
+    meta: RapidsMeta[_, _, _]
+    ): Map[DataType, Set[String]] = {
+    groupedByType.filterKeys {
+      case TimestampType if !TypeChecks.areTimestampsSupported(ZoneId.systemDefault()) ||
```

Comment:
Nit, Extract a method for the following check:
```
          TypeChecks.areTimestampsSupported(ZoneId.systemDefault()) &&
          TypeChecks.areTimestampsSupported(SQLConf.get.sessionLocalTimeZone)
```

Reply:
done